data/lang/module-combat/en.json

```diff
@@ -0,0 +1,346 @@
+   "DENY_1" : {
+      "description" : "",
+      "message" : "Sorry, I need a pilot who has more combat experience."
+   },
+   "DENY_2" : {
+      "description" : "",
+      "message" : "Excuse me sir. I need a real professional."
+   },
+   "DISTANCE" : {
+      "description" : "Distance in space, between two locations",
+      "message" : "Distance:"
+   },
+   "FAILUREMSG_1" : {
+      "description" : "",
+      "message" : "Shame on you. Get out of my sight."
+   },
+   "FAILUREMSG_2" : {
+      "description" : "",
+      "message" : "You are too late! You have caused great losses. You will not be paid!"
+   },
+   "FAILUREMSG_3" : {
+      "description" : "",
+      "message" : "You're a disgrace to all pilots. I cannot pay you!"
+   },
+   "FAILUREMSG_4" : {
+      "description" : "",
+      "message" : "You failed! I will not pay you!"
+   },
+   "GREETING_1" : {
+      "description" : "",
+      "message" : "Hi, I'm {name}."
+   },
+   "GREETING_2" : {
+      "description" : "",
+      "message" : "Hello. My name is {name}."
+   },
+   "GREETING_3" : {
+      "description" : "",
+      "message" : "How are you. I'm {name}."
+   },
+   "GREETING_4" : {
+      "description" : "",
+      "message" : "Greetings! Call me {name}."
+   },
+   "GREETING_5" : {
+      "description" : "",
+      "message" : "Good day. My name is {name}."
+   },
+   "HOW_WILL_I_BE_PAID" : {
+      "description" : "",
+      "message" : "How will I be paid?"
+   },
+   "MANAGER" : {
+      "description" : "",
+      "message" : "Manager"
+   },
+   "MEET_ME_THERE" : {
+      "description" : "",
+      "message" : "Meet me in the {rendezvous} system to finish the job!"
+   },
+   "MILITARY_1" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "Military intelligence believe that a hostile faction has set up a secret listening outpost on {area} in the {system} system ({sectorx}, {sectory}, {sectorz}). The government can not tolerate this and have approved a series of operations to wipe out this facility. The enemy tries to disguise its origin by using only civilian spacecraft types. We do it like this too! So we are willing to hire a mercenary for {cash}. {area} is located {dist} ly from here."
+   },
+   "MILITARY_2" : {
+      "description" : "{area} is an asteroid or planet",
+      "message" : "In response to a case of piracy in the {system} system ({sectorx}, {sectory}, {sectorz}), {dist} ly, the local authorities pay {cash} for a combat mission at {area}."
+   },
+   "MILITARY_3" : {
+      "description" : "",
+      "message" : "The counter intelligence corps needs a civil pilot who asks no questions for a {mission} mission in the {system} system ({sectorx}, {sectory}, {sectorz}), {dist} ly. Pay is {cash}."
+   },
+   "MILITARY_IT_MUST_BE_COMPLETED_BY" : {
+      "description" : "",
+      "message" : "A task force will be assembled to regain control of {area}. So it's important that the mission is successful completed by {date}."
```

Comment:
Nip/Tuck...

> So it's important that the mission is successful completed by {date}."

**It's** important that the mission is **successfully** completed by {date}."